src/java.base/share/classes/java/util/Locale.java

```diff
@@ -2393,7 +2345,13 @@ public boolean equals(Object obj) {
     private static volatile Locale defaultDisplayLocale;
     private static volatile Locale defaultFormatLocale;
 
-    private transient volatile String languageTag;
+    private transient final Supplier<String> languageTag =
```

Comment:
let's use blessed modifiers order
```suggestion
    private final transient Supplier<String> languageTag =
```